scripts/ec2cleanup.py

```diff
@@ -218,8 +220,29 @@ def delete_queues(provider_mgmt, excluded_queues, output):
         logger.exception('Exception in %r', delete_queues.__name__)
 
 
+def delete_s3_buckets(provider_mgmt, bucket_name, output):
+    bucket_list = []
+    provider_name = provider_mgmt.kwargs['name']
+    try:
+        for bucket in provider_mgmt.list_s3_bucket():
+            if bucket_name not in bucket:
+                logger.info("Bucket name '%s' not found in list of buckets", bucket_name)
+            else:
+                if provider_mgmt.delete_s3_bucket(bucket_name=bucket_name):
+                    bucket_list.append([provider_name, bucket_name])
+        logger.info("  Deleted S3 Buckets: %r", bucket_list)
```

Comment:
Required:
This should be after check `if bucket_list`
means we can log like...

```
if bucket_list:
    # generate report
    logger.info("Deleted S3 Buckets: %r", bucket_list)
else:
    logger.info("No %s buckets found for deletion", bucket_name)
```

Reply:
@digitronik Updated.

Reply:
@sbulage, what is this space for ?

Reply:
Removed.